### DISCUSSION TITLE
Different filter lists for different clients?

### DISCUSSION THREAD

**Discussion by user1939** [AdguardTeam/AdGuardHome#6838] [Q&A] [answered]
Hello everyone,

it is possible in client settings to block specific services, but I could not find how to use specific filter lists.

The only option is to 'Block domains using filters and hosts files', but no options to select which to use for specific clients or networks.

I am using custom filtering rules for few specific domains, but I cannot create thousands of records for each client.

Any advice or suggestions would be greatly appreciated.

Thank you.

**Comment by ghost** [comment, marked as answer]
Not through the user interface, however using the `$ctag` modifier should work in the filtering rules.

Example: `||example.org^$ctag=device_pc|device_phone`, which would block access to `example.org` for clients tagged as `device_pc` or `device_phone`.

**Reply by user1939** [reply]
Thank you for your response.
As I have noted, I have been using custom filtering rules, but it is not possible/manageable to create large number of exceptions.

I do not know how exactly block of specific services is done, but I assume that it is by list of domain names which a particular service use. Therefore, I would anticipate that if there was an option to create 'custom service' with a list of specific domain names, it should be possible to block them individually. 

However, as it is, 'custom service' is not available feature and it is not possible to have specific filter list for different clients or networks.

**Comment by ptrsmk** [comment]
Is this on the roadmap at all?  Per client/tag lists.  There are so many reasons for wanting this.

**Reply by smarteshome2020** [reply]
Good question, i'm also need the lists for several Client, Towday it`s only possibe to set to "use global setting".
It will be great to have the posiblitity to selecet "DNS Block-/Allow-List in the Client settings.
